travisciapiprovider.py

```diff
@@ -0,0 +1,16 @@
+import urllib
+import json
+
+# If more functionality is needed from this class, it might be a
+# better decision to use travispy
+class TravisCiApiProvider():
+    host_url = 'https://api.travis-ci.org'
```

Comment:
We'll probably need to make this configurable somehow to we could use file:// urls for tests.